docs/source/reference/builtin-devices.rst

```diff
@@ -40,22 +40,60 @@ This must be sub-classed (like :class:`~ophyd.device.Device`) to be useful.
    ophyd.epics_motor.EpicsMotor
    ophyd.epics_motor.MotorBundle
 
+.. _built_in.EpicsScaler:
+
 EpicsScaler
 -----------
 
 Create an ``EpicsScaler`` to control an EPICS `scaler record
-<http://www.aps.anl.gov/bcda/synApps/std/scalerRecord.html>`_:
+<https://htmlpreview.github.io/?https://github.com/epics-modules/scaler/blob/master/documentation/scalerRecord.html>`_:
 
 .. code-block:: python
 
     from ophyd import EpicsScaler
     scaler = EpicsScaler('XF:28IDC-ES:1{Sclr:1}', name='tth')
 
+Note that :ref:`built_in.ScalerCH` is an alternative ``ophyd`` representation of
+the EPICS `scaler record
```

Comment:
I think sphinx is not happy with a single backtick hanging around.
```suggestion
the EPICS' scaler record
```

Reply:
same thing, not punctuation